timesketch/lib/analyzers/browser_search.py

```diff
@@ -214,16 +223,57 @@ def run(self):
         if simple_counter > 0:
             view = self.sketch.add_view(
                 view_name='Browser Search', analyzer_name=self.NAME,
-                query_string='tag:"browser-search"')
+                query_string='tag:"browser-search"',
+                additional_fields=self._FIELDS_TO_INCLUDE)
             params = {
                 'field': 'search_string',
                 'limit': 20,
             }
-            self.sketch.add_aggregation(
+            agg_obj = self.sketch.add_aggregation(
                 name='Top 20 browser search queries.', agg_name='field_bucket',
```

Comment:
Can you remove the "." from the name? I think it looks nicer without it in the UI. WDYT? :)

Reply:
done